content/deprecated-components/markdown-editor/react/deprecated.mdx

```diff
@@ -0,0 +1,165 @@
+---
+title: Markdown editor
+description: Full-featured Markdown input.
+reactId: markdown_editor
+reactStatus: deprecated
+---
+
+import ReactComponentPage from '~/src/components/react-component-page'
+import { graphql } from "gatsby"
+
+export const query = graphql`
+  query MarkdownEditorDeprecatedQuery($componentId: String! = "markdown_editor", $parentPath: String! = "/deprecated-components/markdown-editor", $status: String! = "deprecated") {
+    ...ReactComponentInfo
+  }
+`
+
+export default function Layout({data, children}) {
+  return <ReactComponentPage data={data} showImport={false} tocItems={[{url: "#import", title: "Import"}, {url: "#examples", title: "Examples"}]}>{children}</ReactComponentPage>
+}
+
+`MarkdownEditor` is a full-featured editor for GitHub Flavored Markdown, with support for:
```

Comment:
I think we can just link to Primer Recipes https://ui.githubapp.com/storybook/?path=/docs/recipes-markdowneditor--readme

Reply:
Is it possible someone is using this component in dotcom and needs to reference the deprecated docs?

Reply:
Not seeing it in [Primer Query](https://musical-adventure-wlr3n3k.pages.github.io/?name=markdowneditor)